### PR TITLE
Selection Sort Implementation in C

### PR DESCRIPTION
**Pull request by byomakeshpanda**
Selection Sort algorithm implementation in C

### CODE REVIEW

**Comment by harshit-sharma-gits** [comment]
You might want to rename your file as `selectionsort.c` as mentioned in #13

**Comment by byomakeshpanda** [comment]
Okay renaming it

File: Algorithms/SelectionSort/selectionsort.c [hunk 1]
```diff
@@ -0,0 +1,56 @@
+#include <stdio.h>
```
Comment (harshit-sharma-gits) [inline Algorithms/SelectionSort/selectionsort.c:1]: Add your name, email in the specifies format mentioned in #13

File: Algorithms/merge_sort.c [hunk 2]
```diff
@@ -0,0 +1,73 @@
+#include <stdio.h>
```
Comment (harshit-sharma-gits) [inline Algorithms/merge_sort.c:1]: Add your name, email in the specifies format mentioned in #13

**Review by harshit-sharma-gits — CHANGES_REQUESTED** [review]
Please check out comments I added. And `mergesort.cpp` was added in a previous PR, so you might want to remove this.

**Pull request closed without merging**